Source/MeleeTrace/Private/MeleeTraceComponent.cpp

```diff
@@ -261,14 +262,18 @@ void UMeleeTraceComponent::InternalStartTrace(const FMeleeTraceInfo& MeleeTraceI
 				MeleeTraceInfo.EndSocketName,
 				NewMeleeTraceInfo.PreviousFrameSampleLocations);
 			OnTraceStart.Broadcast(this, NewMeleeTraceInfo.TraceHandle);
-			return;
+			AddedAny = true;
 		}
 	}
 
-	ensureAlwaysMsgf(false,
-		TEXT("None of the USkeletalMeshComponents contain sockets with names: %s and %s"),
-		*MeleeTraceInfo.StartSocketName.ToString(),
-		*MeleeTraceInfo.EndSocketName.ToString());
+	if (!AddedAny)
+	{
+		UE_LOG(LogTemp,
```

Comment:
I think engine plugins are not allowed to use generic log categories. It would fail to compile and fail the submission to marketplace (fab). I'm happy to accept this PR if you can declare and define a log category for this inside `MeleeTrace.h/cpp` files? Something like `LogMeleeTrace`?

Reply:
Looks like you already have, I just missed it.
Using it now, updated this PR.